eXo/Sources/Controllers/AddDomainVC/CustomPopupVC/CustomPopupViewController.swift

```diff
@@ -10,6 +10,7 @@ import UIKit
 enum ActionHandler {
     case defaultAction
     case delete
+    case store
```

Comment:
the word "store" refers to what ?

Reply:
To handle the popup fonctionality, case store add Install button the popup in order to redirect the user to the Jitsi App on AppStore.

Reply:
Ok fine
Could you update the description of the PR ?

Reply:
Done